data/blog/fourth-wave-of-education-grants.mdx

```diff
@@ -0,0 +1,141 @@
+---
+title: "Fourth Wave of Education Grants"
+date: '2025-04-04'
+tags: ['OpenSats', 'grants', 'bitcoin', 'education', 'wave']
+authors: ['dread', 'arvin', 'julian']
+images: ['/static/images/blog/75-fourth-wave-of-education-grants.jpg']
+draft: false
+summary: "OpenSats is funding three more projects in our education initiative."
+---
+
+We are excited to announce our latest wave of grants from our [Education
+Initiative](/blog/announcing-the-opensats-education-initiative), which is
+dedicated to advancing Bitcoin literacy and community development across the
+globe. These grants represent our ongoing commitment to supporting projects that
+make Bitcoin education more accessible, practical, and useful.
+
+Whether training the next generation of developers in their local languages,
+creating censorship-resistant content, or expanding grassroots curriculum, each
+project in this wave aims to empower individuals and communities with the tools
+and understanding necessary to participate in the growing Bitcoin ecosystem.  
+
+This wave of grants is made possible by the generosity of our donors, whose
+support enables us to fund impactful educational initiatives. Educational
+content produced by these projects will be published under permissive and open
+licenses, ensuring it remains accessible to anyone.
+
+Here are the three projects receiving grants in this wave:
+
+- [Librería de Satoshi](#librería-de-satoshi)
+- [Crack the Orange](#crack-the-orange)
+- [FREE Madeira](#free-madeira)
+
+Each of these projects brings unique value to the global Bitcoin community. We
+are proud to support their efforts and look forward to seeing the impact they
+will have in their respective regions and beyond.
+
+Let's take a closer look at these projects and see how they align with our
+[mission](/mission).
+
+<center>
+    <DonateToGeneralFundButton/>
+</center>
+
+---
+
+### Librería de Satoshi
+
+Librería de Satoshi is a technical school working to make Bitcoin development an
+accessible career path for people across Latin America and the Caribbean. Their
+programs provide real-time, cohort-based technical training in Spanish. Over the
+past three years, Librería de Satoshi has trained over 1,000 people across 26
+countries. Its curriculum covers Bitcoin Script, the Bitcoin protocol, nostr,
+Lightning payments, and bitcoin self-custody. Alumni have gone on to launch
```

Comment:
Consistency flag:

Use of lower case bitcoin in `bitcoin self-custody` vs uppercase in `Bitcoin Script`, `the Bitcoin Protocol`

Reply:
might be nitpicky or might be incorrect, not sure if the 2 uppercase are considered proper nouns?